Add `osu!mania 4K World Cup 2024: Semifinals Recap` news post (#12168)
ppy/osu-wiki: news/2024/2024-09-18-osumania-4k-world-cup-2024-semifinals-recap.md

---
layout: post
title: "osu!mania 4K World Cup 2024: Semifinals Recap"
date: 2024-09-18 21:30:00 +0000
---

> The walls are slowly closing in for the remainder teams, and only the top 6 are making it to the Finals. It's time to lock in, review the previous week, and prepare thy selves for the upcoming showdown of epic proportions.

Reviewer: ```suggestion
The walls are slowly closing in for the remaining teams, and only the top 6 are making it to the Finals. It's time to lock in, review the previous week, and prepare ourselves for the upcoming showdown of epic proportions.
```

![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)

> At this point in the tournament, teams cannot get away with just raw skill, they must watch and analyse their opponents, and make sure they make all the right moves at any given point. Matches slowly turn into masterful showcases of not just skill, but the team's knowledge of the opposing team and their strengths and weaknesses. Each match is played like a delicate dance of tango, where nothing but perfect strategy and execution leads to a victory, with teams constantly looking for any smallest missteps by their opponents.

Reviewer: oo, good intro

```suggestion
At this point in the tournament, teams cannot get away with just raw skill. They must watch and analyse their opponents and make sure they make all the right moves at any given point.

Matches slowly turn into masterful showcases of not just skill, but the team's knowledge of the opposing team and their strengths and weaknesses. Each match is played like a delicate dance of tango, where nothing but perfect strategy and execution leads to victory, with teams constantly looking for any smallest missteps by their opponents.
```

> Let us ponder no more and take a look at the best matches of this week, and see whether teams were doing the tango, or stumbled throughout their matches.

Reviewer: ```suggestion
Let us ponder no more and take a look at the best matches of this week, where we'll see which teams stepped best to the beat, and which teams stumbled to their feet.
```

## Mappool showcase

<style>
    .osu-md--news .audio-player {
        max-width: inherit;
    }
</style>

[**Download the pool here! (212 MB)**](https://osu.ppy.sh/beatmaps/packs/P260)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2246114290&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> With Semifinals comes the second and last time we see two new slots put into the mix, first being Stamina (RC3), with [Black Chocolates](https://osu.ppy.sh/beatmapsets/1433708#mania/3401103) By [Shoegazer](https://osu.ppy.sh/users/2520707). The stamina pick is usually similar to the Jumpstream pick, with the big difference being that it plays differently and goes on for much longer, which is a large reason why it's named Stamina in the first place.

Reviewer: these seem to be skill sets or map types, and not the mappool slots we would typically capitalise

```suggestion
With Semifinals comes the second and last time we see two new slots put into the mix, first being stamina (RC3), with [Black Chocolates](https://osu.ppy.sh/beatmapsets/1433708#mania/3401103) by [Shoegazer](https://osu.ppy.sh/users/2520707). The stamina pick is usually similar to the jumpstream pick, with the big difference being that it plays differently and goes on for much longer, which is a large reason why it's named stamina in the first place.
```

The final new slot is LN4, which is made to contrast LN3, which in this case was [AZISAI](https://osu.ppy.sh/beatmapsets/2246365#mania/4775956) by [lemonguy](https://osu.ppy.sh/users/4693052), being very shield-heavy for this specific round.

> To explain the pool difficulty (and what exactly LN4 is), it's best to let the THREE new originals from this round do the talking!

Reviewer: ```suggestion
To explain the pool difficulty (and what exactly LN4 is), it's best to let the **3 new [originals](/wiki/osu!_originals)** from this round do the talking!
```

[![](/wiki/shared/news/2024-09-18-osumania-4k-world-cup-2024-semifinals-recap/hubris.jpg)](https://osu.ppy.sh/beatmapsets/2246389#mania/4776009)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/364/Songs/kaitendaentai_-_Hubris.mp3">
    </audio>
</div>

[Vortex-](https://osu.ppy.sh/users/4999669) has had a following for a long time, and after going over a year without any new uploads, he makes a return with [Hubris](https://osu.ppy.sh/beatmapsets/2246389#mania/4776009) by [kaitendaentai](https://osu.ppy.sh/beatmaps/artists/364)!

> Taking up the final rice slot, it takes on a similar approach for what we've seen in previous rounds, where it's heavy on 1/4 jumpstream and has scary bursts. The bursts are all over the place, ranging from short 1/6 streams or 1/3 jack-heavy sections. Anyone whose enjoyed a Vortex- chart in the past will be sure to enjoy this one.

Reviewer: ```suggestion
Taking up the final rice slot, it takes on a similar approach for what we've seen in previous rounds, where it's heavy on 1/4 jumpstream and has scary bursts. The bursts are all over the place, ranging from short 1/6 streams or 1/3 jack-heavy sections. Anyone who's enjoyed a Vortex- chart in the past will be sure to enjoy this one.
```

With Akiri already having a large amount of originals in their catalogue, you can expect no less from this one. Being very minijack and jack heavy overall, you're thrown a heavy amount of shields, calling for a great challenge to those with good finger control, especially when having to balance out the 1/8 snaps between some long notes.

[![](/wiki/shared/news/2024-09-18-osumania-4k-world-cup-2024-semifinals-recap/dear_the_night_i_loved.jpg)](https://osu.ppy.sh/beatmapsets/2246380#mania/4775997)

<div class="osu-md__paragraph">
    <audio controls>
        <source src="https://assets.ppy.sh/artists/345/Songs/Kabocha_feat._Aitsuki_Nakuru_-_Dear_The_Night_I_Loved.mp3">
    </audio>
</div>

Last, but CERTAINLY not least, [Kabocha](https://osu.ppy.sh/beatmaps/artists/345) and incredible vocalist [Aitsuki Nakuru](https://osu.ppy.sh/beatmaps/artists/127) bring [Dear The Night I Loved](https://osu.ppy.sh/beatmapsets/2246380#mania/4775997) charted by [-mint-](https://osu.ppy.sh/users/8976576) and [MyZterioN-](https://osu.ppy.sh/users/8976576)!

> If you're at all familiar with their song Lilith, you're familiar with this. As described earlier with this being LN4, it's much more stream/snap focused, leading to smooth patterns that can have a heavy toll on your accuracy due to the nature of how staggered (just about) every release is. All long note players are sure to enjoy this one.

Reviewer: ```suggestion
If you're at all familiar with their song *Lilith*, you're familiar with this one. This is the aforementioned LN4, and it's much more stream/snap focused, leading to smooth patterns that can have a heavy toll on your accuracy due to the nature of how staggered (just about) every release is. All long note players are sure to enjoy this one.
```

## Quarterfinals results

> The table keeps getting smaller, but the matches contained within keep getting spicier and spicier.

Reviewer: bit more commentary

```suggestion
The table keeps getting smaller, but the matches keep getting spicier.

In these tables, Team A is the higher seed, or in other words performed better in the Qualifiers, yet at this level, such orderings determine less of the outcome.
```

- [trapman - minefield outlaw](https://osu.ppy.sh/beatmapsets/2246339#mania/4775913) ([guden](https://osu.ppy.sh/users/11626065), RC4) @ [13:53](https://www.twitch.tv/videos/2250787141?t=00h13m53s) - With the score being 1-0, it was now Hong Kong’s turn to pick, and their pick was RC4, a stamina draining chordjack map with tricky patterns to break combo on. While Hong Kong was able to stay in the lead for a good chunk of the map, due to combo breaks near the end of the map, the Philippines was able to take the breakpoint.
- [YOASOBI - Idol](https://osu.ppy.sh/beatmapsets/2246343#mania/4775926) ([0DZ0](https://osu.ppy.sh/users/28156707), RC6) @ [18:29](https://www.twitch.tv/videos/2250787141?t=00h18m29s) - Hong Kong needed revenge from the Philippines if they wanted to stay alive and tie the breakpoints, and in an interesting turn of events, the Philippines chose RC6, a vocal stream dump map. This pick ended up being way too close with the score difference not even going over 10k, and in the end, Hong Kong managed to get their sweet revenge and take a breakpoint from the Philippines

> - [Tatsh - Joker -Vocaloid ver.-](https://osu.ppy.sh/beatmapsets/2246033#mania/4775287) ([Valedict](https://osu.ppy.sh/users/2537082), RC1) @ [58:25](https://www.twitch.tv/videos/2250787141?t=00h58m25s) - It’s now match point for the Philippines with the score being 6-3. It was time for Hong Kong to make their pick, and their pick was RC1, a dense stream map with practically zero breaks. Hong Kong needed to win if they wanted to continue having a fighting chance at taking this to the tiebreaker. All was going well, until the combos broke from the Hong Kong side at the end, resulting in the Philippines taking the breakpoint, and winning the match all together.

Reviewer: ```suggestion
- [Tatsh - Joker -Vocaloid ver.-](https://osu.ppy.sh/beatmapsets/2246033#mania/4775287) ([Valedict](https://osu.ppy.sh/users/2537082), RC1) @ [58:25](https://www.twitch.tv/videos/2250787141?t=00h58m25s) - It's now match point for the Philippines at 6–3. It was time for Hong Kong to make their pick, and their pick was RC1, a dense stream map with practically zero breaks. Hong Kong needed to win if they wanted to continue having a fighting chance at taking this to the tiebreaker. All was going well, until the combos broke from the Hong Kong side at the end, resulting in the Philippines winning the match.
```

> Well… that was certainly a match you could say. However, we will have to say our goodbyes to the Hong Kong team as they are now officially out of the tournament. Your underrated performances that shocked many viewers will not be forgotten, we hope they will place better next year. As for the Philippines, they will be facing up against the almighty South Korea team who has previously dominated the osu!mania World Cup, we wish them the very best of luck tackling down a beast of a team.

Reviewer: ```suggestion
Well... that was certainly a match. However, we will have to say our goodbyes to the Hong Kong team as they are now officially out of the tournament. Your underrated performances that shocked many viewers will not be forgotten, and we hope for a better run next year. As for the Philippines, they will be facing up against the almighty South Korean team which has previously dominated the osu!mania World Cup. We wish them the very best of luck.
```

- [t+pazolite - Beatboxer VS Trackmaker feat. KAJI & Kohey](https://osu.ppy.sh/beatmapsets/2246368#mania/4775966) ([AlexDunk](https://osu.ppy.sh/users/9194799), SV2) @ [01:01:12](https://www.twitch.tv/videos/2251756726?t=01h01m12s) - China was on a roll after winning their first break point, making the score 3–6 in their favour. After much consideration, Japan picked SV2, a map that China had already played the day before, in hopes of turning the tides. [CrewK](https://osu.ppy.sh/users/11488604), considered by some as the best player in the world, scored 997,336 points and was ahead of the pack by a lot, however that couldn't save Japan as China's team cobbled together an amazing team score of 2,956,661 team score, securing themselves the map, and a spot in the Finals.

As seed #13, it was already a good result for Japan to arrive to the top 8. Now it is the time to congratulate China for stepping into Finals after a 6 year-long effort. Next week they will be facing another familiar team — Chile, which is a match that cannot be missed.

## Score highlights

> The third to last round ended a few days ago, and with that, the finals grow ever so closer. To the layman, the maps are starting to become nigh unreadable. Yet that's not stopping our players from pulling up Herculean stunts such as their scores suggest!

Reviewer: ```suggestion
The third to last round ended a few days ago, and with that, the finals grow ever closer. To the layman, the maps are starting to become nigh unreadable, but that has yet to stop our players from pulling up Herculean scores!
```

> - It's no surprise to see ::{ flag=CL }:: [myucchi](https://osu.ppy.sh/users/10072733) being highlighted again. Their performance throughout the tournament has simply been stellar. Their play on [yuikonnu - Uchouten Vivace (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2246331#mania/4775902) by [Miaurichesu](https://osu.ppy.sh/users/10055648) (RC2) isn't any different however, achieving a score of 999,649 [against the United States](https://www.twitch.tv/videos/2251402252).

Reviewer: ```suggestion
- It's no surprise to see ::{ flag=CL }:: [myucchi](https://osu.ppy.sh/users/10072733) highlighted again. Their performance throughout the tournament has simply been stellar. Their play on [yuikonnu - Uchouten Vivace (Cut Ver.)](https://osu.ppy.sh/beatmapsets/2246331#mania/4775902) by [Miaurichesu](https://osu.ppy.sh/users/10055648) (RC2) isn't any different however, achieving a score of 999,649 [against the United States](https://www.twitch.tv/videos/2251402252).
```

> - ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) may be a new name being highlighted, but the community already knows what they're capable of. They managed to score one of the highest combined total scores from three separate LN picks. Their peak performance was seen with their score of 995,993 in [Hino Isuka - Dreamin' attraction!!](https://osu.ppy.sh/beatmapsets/2246225#mania/4775650) (LN2) by [Toaph Daddy](https://osu.ppy.sh/users/7616811) [versus South Korea](https://www.twitch.tv/videos/2251727183).

Reviewer: ```suggestion
- ::{ flag=TH }:: [--Pavin--](https://osu.ppy.sh/users/12046267) may be a new name in the spotlight, but the community already knows what they're capable of. They managed to score one of the highest combined total scores from three separate LN picks. Their peak performance was seen with their score of 995,993 in [Hino Isuka - Dreamin' attraction!!](https://osu.ppy.sh/beatmapsets/2246225#mania/4775650) (LN2) by [Toaph Daddy](https://osu.ppy.sh/users/7616811) [versus South Korea](https://www.twitch.tv/videos/2251727183).
```

> We're nearing the very last stretch of the tournament. As both Finals and Grand Finals loom by, we're reaching the apex difficulty of this year's tournament. But will the scores of a few be able to turn the tides in favour of their team? We'll know the answer this week!
> 
> ---

Reviewer: ```suggestion
---

We're nearing the very last stretch of the tournament. As both Finals and Grand Finals loom by, we're reaching the apex difficulty of this year's tournament. Will the scores of a few be able to turn the tides in favour of their team? We'll know the answer this weekend!
```

> The tournament is slowly coming to a close, but that doesn't mean it's too late to join the community on the [osu!live Twitch channel](https://www.twitch.tv/osulive) and cheer on your favourite teams live! If you need help choosing which match to watch, the [wiki page](/wiki/Tournaments/MWC/2024_4K) contains the full schedule, along with any other details you might require to fully immerse yourself in the key smashing showdown.

Reviewer: ```suggestion
And even though the tournament is slowly coming to a close, that doesn't mean it's too late to join the community on the [osu!live Twitch channel](https://www.twitch.tv/osulive) and cheer on your favourite teams live! If you need help choosing which matches to watch, the [wiki page](/wiki/Tournaments/MWC/2024_4K) contains the full schedule, along with any other details you might require to fully immerse yourself in the key smashing showdown.
```

| South Korea ::{ flag=KR }:: | 1 | **7** | ::{ flag=TH }:: **Thailand** |
| **Philippines** ::{ flag=PH }:: | **7** | 2 | ::{ flag=MY }:: Malaysia |
| Japan ::{ flag=JP }:: | 3 | **7** | ::{ flag=CN }:: **China** |

## Match highlights

### ::{ flag=PH }:: Philippines vs. ::{ flag=HK }:: Hong Kong

[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115486378)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2250787141&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

The losers' bracket is a scary position to be in, especially late into the tournament. The Philippines got here thanks to Chile just last round, losing against them 6–2. As for team Hong Kong, they've been clawing their way up the losers' bracket with a clutch victory against the United Kingdom, winning the last four picks when they needed it most.

As both Philippines and Hong Kong entered the World Cup stage, viewers knew that this was going to be one hell of a match.

> - [trapman - minefield outlaw](https://osu.ppy.sh/beatmapsets/2246339#mania/4775913) ([guden](https://osu.ppy.sh/users/11626065), RC4) @ [13:53](https://www.twitch.tv/videos/2250787141?t=00h13m53s) - 1–0, Hong Kong’s turn to pick, and their pick was RC4, a stamina-draining chordjack map with tricky patterns waiting to break combos. While Hong Kong was able to stay in the lead for a good chunk of the map, a few combo breaks near the end of the map let the Philippines take the break point.

Reviewer: ```suggestion
- [trapman - minefield outlaw](https://osu.ppy.sh/beatmapsets/2246339#mania/4775913) ([guden](https://osu.ppy.sh/users/11626065), RC4) @ [13:53](https://www.twitch.tv/videos/2250787141?t=00h13m53s) - 1–0, Hong Kong's turn to pick, and their pick was RC4, a stamina-draining chordjack map with tricky patterns waiting to break combos. While Hong Kong was able to stay in the lead for a good chunk of the map, a few combo breaks near the end of the map let the Philippines take the break point.
```

> - [YOASOBI - Idol](https://osu.ppy.sh/beatmapsets/2246343#mania/4775926) ([0DZ0](https://osu.ppy.sh/users/28156707), RC6) @ [18:29](https://www.twitch.tv/videos/2250787141?t=00h18m29s) - Hong Kong needed revenge from the Philippines if they wanted to stay alive, and in an interesting turn of events, the Philippines chose RC6, a vocal stream dump map. This pick ended up being way too close with a score difference not even 10k, and in the end, Hong Kong managed to respond in kind with a break point.

Reviewer: ```suggestion
- [YOASOBI - Idol](https://osu.ppy.sh/beatmapsets/2246343#mania/4775926) ([0DZ0](https://osu.ppy.sh/users/28156707), RC6) @ [18:29](https://www.twitch.tv/videos/2250787141?t=00h18m29s) - Hong Kong needed revenge from the Philippines if they wanted to stay alive. In an interesting turn of events, the Philippines brazenly chose RC6, a vocal stream dump map, despite Hong Kong's exceptional rice skills. This pick ended up being way too close with a score difference not even 10k, and in the end, Hong Kong managed to respond in kind with a break point.
```

Well... that was certainly a match. However, we will have to say our goodbyes to the Hong Kong team as they are now officially out of the tournament. Your underrated performances that shocked many viewers will not be forgotten, and we hope for a better run next year. As for the Philippines, they will be facing up against the almighty South Korean team which has previously dominated the osu!mania World Cup. We wish them the very best of luck.

### ::{ flag=JP }:: Japan vs. ::{ flag=CN }:: China

[**You can view the match information here!**](https://osu.ppy.sh/community/matches/115504471)

<div align="center" class="osu-md__paragraph">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2251756726&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
</div>

> In 2022's MWC 4K , China was beaten by Japan in Semifinals, and by coincidence, they met in same round again this year. This time both teams changed a lot, China just swept Vietnam in round 1, while Japan successfully defeated Singapore without much trouble. It is well-known that Japan has a strong roster with impressive regular skill, and China players shows their great consistency in every skill set. Both teams are also strong in SV, and that ended up being the key to winning this match.

Reviewer: ```suggestion
In 2022's MWC 4K, China was beaten by Japan in the Semifinals, and by coincidence, they met in same round again this year. Both teams have changed a lot since: China just swept Vietnam in round 1, while Japan successfully defeated Singapore without much trouble. It is well known that Japan has a strong roster with impressive raw skill, and that Chinese players have great consistency in every skill set. Both teams are also strong in SV, and that ended up being the key to winning this match.
```

- [Terminal 11 - Cold Heart Remix](https://osu.ppy.sh/beatmapsets/2246337#mania/4775909) ([Pizza69](https://osu.ppy.sh/users/25285704), RC5) @ [47:16](https://www.twitch.tv/videos/2251756726?t=00h47m16s) - The match was 7 maps in without any break points, making the score 3–4 for China. Japan, after picking RC 2 through 4, opted to stick to what they do best, and picked RC5. Unfortunately for Japan, China was very strong on this pick, and a miss by ::{ flag=JP }:: [AFOTHER](https://osu.ppy.sh/users/10242062) near the end of the map sealed the deal and granted China their first break point, creating a point gap between the two teams.

> - [t+pazolite - Beatboxer VS Trackmaker feat. KAJI & Kohey](https://osu.ppy.sh/beatmapsets/2246368#mania/4775966) ([AlexDunk](https://osu.ppy.sh/users/9194799), SV2) @ [01:01:12](https://www.twitch.tv/videos/2251756726?t=01h01m12s) - China was on a roll after winning their first break point, making the score 3–6 in their favour. After much consideration, Japan picked SV2, a map that China had already played the day before, in hopes of turning the tides. [CrewK](https://osu.ppy.sh/users/11488604), considered by some as the best player in the world, scored 997,336 points and was ahead of the pack by a lot, however that couldn't save Japan as China's team cobbled together an amazing team score of 2,956,661 team score, securing themselves the map, and a spot in the Finals.

Reviewer: ```suggestion
- [t+pazolite - Beatboxer VS Trackmaker feat. KAJI & Kohey](https://osu.ppy.sh/beatmapsets/2246368#mania/4775966) ([AlexDunk](https://osu.ppy.sh/users/9194799), SV2) @ [01:01:12](https://www.twitch.tv/videos/2251756726?t=01h01m12s) - China was on a roll after winning their first break point, making the score 3–6 in their favour. After much consideration, Japan picked SV2, a map that China had already played the day before, in hopes of turning the tides. [CrewK](https://osu.ppy.sh/users/11488604), considered by some as the best player in the world, scored 997,336 points and was ahead of the pack by a lot. However, that couldn't save Japan as China's team cobbled together an amazing team score of 2,956,661, securing themselves the map.
```

> As seed #13, it was already a good result for Japan to arrive to the top 8. Now it is the time to congratulate China for stepping into Finals after a 6 year-long effort. Next week they will be facing another familiar team — Chile, which is a match that cannot be missed.

Reviewer: ```suggestion
As seed #13, it was already a good result for Japan to arrive to the top 8. Now it is the time to congratulate China for stepping into the Finals after a 6-year-long effort. Next week, they will be facing another familiar team, Chile, which is a match that cannot be missed.
```
